libs/blocks/table/table.css

```diff
@@ -323,6 +331,11 @@ header.global-navigation {
     margin: 0 30px;
   }
 
+  .table:not(.merch) .row .section-head-title,
+  .table:not(.merch) .row .section-row-title {
+    border-right: 1px solid var(--border-color);
```

Comment:
Is this being reflected for RTL?

Reply:
This is for fixing mobile issue:
<img width="714" alt="image" src="https://github.com/adobecom/milo/assets/55804872/273f514b-d75e-4563-8750-d89808788eed">